src/test/beatgridtest.cpp

```diff
@@ -26,12 +26,12 @@ TEST(BeatGridTest, Scale) {
     int sampleRate = 44100;
     TrackPointer pTrack = newTrack(sampleRate);
 
-    double bpm = 60.0;
+    const auto bpm = 60.0;
```

Comment:
why not make this `mixxx::Bpm`?

Reply:
Because then we'd have to convert everywhere. A later PR will change this.

Reply:
Ok, I guess I only counted the three lines where this would save on diff but didn't see any of the occurrences where this would bloat the diff.

Reply:
All the comparisons below :D